applications/asset_tracker/prj_nrf9160_pca20035ns.conf

```diff
@@ -74,6 +74,9 @@ CONFIG_GPS_CONTROL_FIX_CHECK_INTERVAL=30
 CONFIG_GPS_CONTROL_FIX_TRY_TIME=360
 CONFIG_GPS_CONTROL_PSM_DISABLE_ON_STOP=n
 
+CONFIG_GPS_START_ON_MOTION=y
+
```

Comment:
One empty line is enough. I also think it there's no need to separate this config from other GPS configs with an empty line.